legend-engine-pure-ide-light/src/main/java/org/finos/legend/engine/ide/PureIDELight.java

```diff
@@ -101,14 +101,14 @@ private static void enableEngineIntegration()
     {
 
         System.setProperty("alloy.test.server.host", "127.0.0.1");
-        System.setProperty("alloy.test.server.port", "6060");
+        System.setProperty("alloy.test.server.port", "6300");
```

Comment:
Thought your previous instructions were to make it 6060 for enabling local debug??

Reply:
Indeed, but since then they have changed the legend-engine server port to 6300 ( from 6000) so this makes it more inline with finos/master https://github.com/ghub-real/legend-engine/commit/0f37e3610c9dfcefed3faf82b05b2a15a2ea4348